retrodeck.sh

```diff
@@ -89,20 +89,27 @@ standalones_init() {
     dir_prep "$rdhome/bios/switch/keys" "/var/data/yuzu/keys"
     # initializing the firmware folder
     dir_prep "$rdhome/bios/switch/registered" "/var/data/yuzu/nand/system/Contents/registered"
+    # initializing the save folders
+    dir_prep "$rdhome/saves/switch/yuzu/nand" "/var/data/yuzu/nand"
+    dir_prep "$rdhome/saves/switch/yuzy/sdmc" "/var/data/yuzu/sdmc"
     # configuring Yuzu
     dir_prep "$rdhome/.logs/yuzu" "/var/data/yuzu/log"
     mkdir -pv /var/config/yuzu/
+    if [ $overwrite_configs = true ]; then # Initializing config after major version update
```

Comment:
This is gold, but I would add them in 0.6.0b as I consider this an enhancement more than a bug fix in this case.

Reply:
I've reverted these commits, we can rethink this functionality later.